definitions/divorce/json/CaseType.json

```diff
@@ -2,7 +2,7 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "DIVORCE",
-    "Name": "Divorce case - v113.18",
+    "Name": "Divorce case - v113.19",
```

Comment:
You're going to clash with my v113.19 dammit 😂
https://github.com/hmcts/div-ccd-definitions/pull/13/files#diff-94444d93f934f87c887953da988c8b6fR5

Reply:
Nevermind, I guess whoever merges first will resolve the conflict and up the version..

Reply:
race is on. Except I'm doing some more changes now :( Hold reviews please.

Reply:
Beat you to it mate, sorry :)